daemon/images.go

```diff
@@ -41,28 +42,35 @@ func (d *Daemon) pollForNewImages(logger log.Logger) {
 	changes := &update.Automated{}
 	for _, service := range services {
 		for _, container := range service.ContainersOrNil() {
-			logger := log.With(logger, "service", service.ID, "container", container.Name, "currentimage", container.Image)
-
 			currentImageID := container.Image
-			if err != nil {
-				logger.Log("error", err)
-				continue
-			}
```

Comment:
that `err != nil branch` was dead code, right?

Reply:
Right; it used to parse the image string, but that gets done earlier now.